### PR TITLE
Create Caroussel

### PR DESCRIPTION
**Pull request by Franck59840**
Create components Slider  (Slider Data , SliderImage and Slider Css)

### CODE REVIEW

**Review by Vjerem — DISMISSED** [review]

File: src/components/slider/ImageSlider.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+import SliderData from './SliderData';
+
+const ImageSlider = ({ slides }) => {
+  const [current, setCurrent] = useState(0);
+  const { length } = slides;
+
+  const nextSlide = () => {
+    setCurrent(current === length - 1 ? 0 : current + 1);
```
Comment (LoicBrassart) [inline src/components/slider/ImageSlider.jsx]: Use instead: `setCurrent((current+1)%length)`

File: src/components/slider/ImageSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  };
+
+  const prevSlide = () => {
```
Comment (LoicBrassart) [inline src/components/slider/ImageSlider.jsx]: Same

File: src/components/slider/SliderData.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  {
+    image: ImageFive,
+  },
+  {
+    image: ImageSix,
+  },
+  {
+    image: ImageSeven,
+  },
```
Comment (LoicBrassart) [inline src/components/slider/SliderData.jsx]: I don't get why you use objects to store simple data
Why not something like `[imageOne, imageTwo, imageThree, ...]` ? Do you plan on adding more data to these objects ?

File: src/components/slider/ImageSlider.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    setCurrent(current === 0 ? length - 1 : current - 1);
+  };
+
+  if (!Array.isArray(slides) || slides.length <= 0) {
+    return null;
+  }
```
Comment (LoicBrassart) [inline src/components/slider/ImageSlider.jsx]: - `slides` should be a string according to your propTypes, why do you test whether it's an array ?
- You could remove the whole condition and do something like:
```
return slides.length &&     <section className="slider">[...]</section
```

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some things to fix (or at least explain to me)

**Review by KeerhemFR — DISMISSED** [review]
Seems fine to me

**Review by KeerhemFR — APPROVED** [review]
Seems good to me

**Review by Hacene010 — APPROVED** [review]
seems ok

**Pull request closed without merging**